gpumon/gpumon_client.ml

```diff
@@ -0,0 +1,31 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open Gpumon_interface
+open Xcp_client
+
+let json_url () = "file:" ^ json_path
```

Comment:
This doesn't look like a file URL. I expected to see `"file://"^json_path`. https://en.wikipedia.org/wiki/File_URI_scheme#Unix